easybuild/easyblocks/b/bowtie2.py

```diff
@@ -16,12 +16,13 @@
 @author: Fotis Georgatos (Uni.Lu)
 @author: Kenneth Hoste (Ghent University)
 """
-
+from distutils.version import LooseVersion
 import os
 import shutil
 
 from easybuild.easyblocks.generic.configuremake import ConfigureMake
 from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.filetools import copy_file
 
 
 class EB_Bowtie2(ConfigureMake):
```

Comment:
We have a whole bunch of blocks that already do this (deriving from `MakeCp` and filling in the `files_to_copy`) so I don't see why this should be any different. Any future enhancements made to `MakeCp` will also need to be ported to this block. 

Now you have to take the action of letting the configure_step be `pass` while in the MakeCp case you need to specify that `files_to_copy` is not mandatory in the easyconfig. So I don't see much difference in that regard.